test/index.test.ts

```diff
@@ -1,8 +1,85 @@
 import { expect, it, describe } from "vitest";
-import {} from "../src";
+import {
+  bold,
+  blockquote,
+  boldAndItalic,
+  codeBlock,
+  heading,
+  hr,
+  image,
+  italic,
+  link,
+  list,
+  strikethrough,
+  table
+} from "../src";
 
 describe("omark", () => {
-  it.todo("pass", () => {
-    expect(true).toBe(true);
+  it("bold", () => {
+    expect(bold("Hello, World!")).toBe("**Hello, World!**");
   });
+
+  it("blockquote", () => {
+    expect(blockquote("Hello, World!")).toBe("> Hello, World!");
+  });
+
+  it("boldAndItalic", () => {
+    expect(boldAndItalic("Hello, World!")).toBe("***Hello, World!***");
+  });
+
+  it("coldBlock", () => {
+    expect(codeBlock('console.log("Hello, World!");', "js")).toBe(
+      '```js\nconsole.log("Hello, World!");\n```',
+    );
+  });
+
+  it("heading", () => {
+    expect(heading("Hello, World!", 1)).toBe("\n# Hello, World!\n");
```

Comment:
This can be handled dynamically:

```ts
describe("heading", () => {
    for (const level of [1, 2, 3, 4, 5, 6]) {
      it(`${"#".repeat(level)} Heading`, () => {
        expect(heading("Hello, World!", level)).toBe(
          `\n${"#".repeat(level)} Hello, World!\n`,
        );
      });
    }
  });
  ```

Reply:
It is probably over-complexity.